tests/image-stream/utils.spec.js

```diff
@@ -26,44 +26,34 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
+    it('should resolve application/x-bzip2 for a bz2 archive', function() {
```

Comment:
Not necessarily wrong, the failures would just take longer, and manifest in timeouts rather than having a little more helpful stack trace and error message.
Yes, I think there are still a few tests where this is the case, can do a check & follow up PR on those.